src/components/FeedPage.vue

```diff
@@ -24,21 +24,26 @@
 
     <hr />
 
-    <div class="video-grid">
+    <div v-if="videos.length != 0" class="video-grid">
```

Comment:
Maybe a different condition should be checked? Ex: what if a user isn't subscribed to any channel (maybe they forgot to login). It will just seem like the page is loading forever.

Reply:
True, good point!